CONTRIBUTING.md

```diff
@@ -18,6 +18,7 @@ Storybook - [Storybook Documentation](https://storybook.js.org/)
 
 ## How to contribute
 
+0. Please check the [coding standards](https://github.com/AccessibleForAll/AccessibleWebDev/blob/main/CODING_STANDARDS.md) page before start contributing.
```

Comment:
I didn't come up with a good position for this. Please add some suggestions.

Reply:
Maybe put it as point 2 so they raise an issue first and ask to be assigned, then look at the standards